tests/fixtures/spec-examples-by-section.json

```diff
@@ -13,9 +13,9 @@
        "path"       : "/foo/bar",
        "list"       : ["red", "green", "blue"],
        "keys"       : { "semi" : ";", "dot" : ".", "comma" : ","},
-       "v"          : "6",
-       "x"          : "1024",
-       "y"          : "768",
+       "v"          : 6,
+       "x"          : 1024,
+       "y"          : 768,
```

Comment:
These are imported from a different project, not generated here and I don't modify them

Reply:
To be clear, everything in `tests/fixtures` is from the folks who wrote the RFC in the first place to be used to test projects like this one. I'm not going to edit this

Reply:
https://github.com/uri-templates/uritemplate-test